website/docs/r/cloud_gcp_integrations.html.markdown

```diff
@@ -130,6 +130,7 @@ The following arguments are supported:
 * `functions` - (Optional) Functions integration. See [Integration blocks](#integration-blocks) below for details.
 * `interconnect` - (Optional) Interconnect integration. See [Integration blocks](#integration-blocks) below for details.
 * `kubernetes` - (Optional) Kubernetes integration. See [Integration blocks](#integration-blocks) below for details.
+* `load_balancing` - (Optional) Load Balancing integration. See [Integration blocks](#integration-blocks) below for details.
```

Comment:
Note for the reviewer: 
Added `load_balancing` to the list of arguments as it is supported and is also part of the example, but is missing in this list.